test/jasmine/tests/svg_text_utils_test.js

```diff
@@ -105,6 +105,21 @@ describe('svg+text utils', function() {
             assertAnchorLink(node, 'mailto:support@plot.ly');
         });
 
+        it('wrap XSS attacks in href', function() {
```

Comment:
@scjody 

The previous commit https://github.com/plotly/plotly.js/pull/736/commits/d58d3bee92cbed88e2da250753d7dcb374d42e1d also appears to have fix the XSS issue discovered in https://github.com/plotly/streambed/issues/7056 - would you mind reviewing this?